### DISCUSSION TITLE
Getting started: navigating to /cp results in Not Found

### DISCUSSION THREAD

**Discussion by JeroenRood** [statamic/cms#8511] [Troubleshooting] [answered]
While following the Quick Start Guide on Statamic Docs, I navigated my way through setting up a project. So far, I managed to install Statamic 4 through Composer, and set up XAMPP to have an additional VirtualHost for my project's directory. I also created a super admin, as recommended by the docs. So far everything seems to work like a breeze. This is the current state of the project:

Platform: Windows 11
Statamic Version (derived from `composer.json`): ^4.0
Project name: dutch-patches
VirtualHost config:
```
<VirtualHost *:80>
    ServerAdmin <some-email-i-chose@host.tld>
    DocumentRoot "C:/a/specific/path/to/the/project/dutch-patches/public"
    ServerName dutchpatches.local
    ErrorLog "logs/dutchpatches.local-error.log"
    CustomLog "logs/dutchpatches.local-access.log" common
    <Directory "C:/a/specific/path/to/the/project/dutch-patches/">
        Require all granted
    </Directory>
</VirtualHost>
```

As a result, I managed to open up `http://dutchpatches.local/` in the browser, to show the `Welcome to your brand new Statamic site!` page. That works.

But when I navigate to `http://dutchpatches.local/cp`, Apache shows me a Not Found page.

I am not sure what I am doing wrong. Is this related to me following the steps wrongly? Or might it be some issue with my XAMPP setup? I do suspect that the VirtualHost is not set up correctly, but in light of a Statamic project, I do not know what might be a good `DocumentRoot` or `<directory />`.

Hopefully, someone can give me some pointers on how to solve this issue, so that I can successfully set up my Statamic project and the environment in XAMPP.

**Comment by JeroenRood** [comment, marked as answer]
As it turns out, I was trying too hard getting it to work on XAMPP, not using htdocs while trying to set up a VirtualHost. In fact, I now learned that Laravel can set up a server in-situ in the project. All I had to do is `cd` into the project directory and run `php artisan serve`. 

Now I can just navigate to `127.0.0.1:8000`, and still get the `Welcome to your brand new Statamic site!` page. Now, navigating to `/cp` works as well.

**Reply by duncanmcclean** [reply]
Glad you managed to get it sorted! I've marked this comment as the 'answer'.
